pkgs/tools/system/xmousepasteblock/default.nix

```diff
@@ -0,0 +1,30 @@
+{ xorg, stdenv, fetchgit, libev, pkgconfig }:
+
+stdenv.mkDerivation {
+  name = "XMousePasteBlock-1.0";
+  src = fetchgit {
+    sha256 = "0vidckfp277cg2gsww8a8q5b18m10iy4ppyp2qipr89771nrcmns";
+    url = "https://github.com/milaq/XMousePasteBlock";
+    rev = "ef7887d75a38628b7c7990c426297beeb639b706";
+  };
+  preBuild = ''
+    substituteInPlace Makefile \
+      --replace '/usr' "''${out}"
+  '';
+ buildInputs = (with xorg;
+  [ libX11
+    libXext
+    libXi
+  ]) ++ [ libev pkgconfig ];
```

Comment:
I don't think you need to distinct arrays.

Reply:
I think `pkgconfig` should be set in `nativeBuildInputs` instead of `buildInputs`